### PR TITLE
[berkeley] Fix LFS errors + compatible merge

### PR DESCRIPTION
**Pull request by mrmr1993**
This PR fixes the diff errors against berkeley PRs by checking the relevant files into git LFS.

### CODE REVIEW

**Comment by mrmr1993** [comment]
!ci-build-me

File: src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml [hunk 1]
```diff
@@ -535,7 +535,8 @@ let setup_daemon logger =
   and simplified_node_stats =
     flag "--simplified-node-stats"
       ~aliases:[ "simplified-node-stats" ]
-      no_arg ~doc:"whether to report simplified node stats (default: false)"
+      (optional_with_default true bool)
```
Comment (emberian) [inline src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml:538]: no? wrong PR?

Comment (mrmr1993) [reply src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml:538]: It came from the merge through of the `compatible` PR :man_shrugging:

Comment (emberian) [reply src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml:538]: 👍🏼

**Review by emberian — APPROVED** [review]

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!approved-for-mainnet

**Review by dkijania — APPROVED** [review]

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
Force-pushed because CI isn't happy still :@

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!approved-for-mainnet

**Pull request merged**